src/test/java/com/ays/AbstractUnitTest.java

```diff
@@ -2,17 +2,12 @@
 
 import com.ays.auth.model.AysToken;
 import org.junit.jupiter.api.extension.ExtendWith;
-import org.junit.runner.RunWith;
-import org.mockito.junit.MockitoJUnitRunner;
 import org.mockito.junit.jupiter.MockitoExtension;
 import org.mockito.junit.jupiter.MockitoSettings;
 import org.mockito.quality.Strictness;
-import org.springframework.test.context.ActiveProfiles;
 
 @ExtendWith(MockitoExtension.class)
-@RunWith(MockitoJUnitRunner.class)
 @MockitoSettings(strictness = Strictness.LENIENT)
-@ActiveProfiles(value = "test")
 public abstract class AbstractUnitTest {
```

Comment:
Controller ve Repository'a eklediğimiz için buraya da  `extends AbstractTestContainerConfiguration ` ekleyebilir miyiz?

`public abstract class AbstractUnitTest extends AbstractTestContainerConfiguration`

Reply:
Aslında Unit Test sürecinde container yapısına ihtiyaç bulunmuyor çünkü bu katmanda yapılan testlerde uygulama ayaklandırılmıyor, scriptler çalıştırılmıyor. Sadece Integration Testlerin yapıldığı katmanlarda bu konfigürasyonu kullanmalıyız, yani Repository ve Controller katmanlarında yazdığımız testlerde kullanmamız yeterli oluyor.

Reply:
`AbstractTestContainerConfiguration` buna gerek yoksa sorun yok.